lib/ansible/modules/cloud/ovh/ovh_monthly_billing.py

```diff
@@ -34,22 +34,40 @@
         type: str
         description:
             - ID of the instance, get it with U(https://api.ovh.com/console/#/cloud/project/%7BserviceName%7D/instance#GET)
+    endpoint:
+        type: str
+        description:
+            - The endpoint to use (for instance ovh-eu)
+    application_key:
+        type: str
+        description:
+            - The applicationKey to use
+    application_secret:
+        type: str
+        description:
+            - The application secret to use
+    consumer_key:
+        type: str
+        description:
+            - The consumer key to use
 '''
 
 EXAMPLES = '''
-# basic
+# basic usage, using auth from /etc/ovh.conf
```

Comment:
does this example still work since we have parameters for credentials now ?

Reply:
Yes it does, parameters for credentials are optional, if absent, ovh lib will look in /etc/ovh.conf or in some environment variables. https://github.com/ovh/python-ovh#configuration

Reply:
OK